docs/site/themes/template/layouts/partials/use-cases.html

```diff
@@ -12,7 +12,7 @@ <h3>Learn</h3>
                 </div>
             </div>
         </a>
```

Comment:
This section talks about a hands-on lab, not the blog. So if this should point to the blog then the copy is incorrect.

Reply:
this is talking about TCE as an environment that people can use to learn in a hands-on way.  It is not referring to a HOL

Reply:
The copy still doesn't refer to the blog though, and the blogs don't focus on hands-on learning, so I'm unclear how the copy fits here.